src/ExperimentalFunds.mo

```diff
@@ -0,0 +1,54 @@
+/// Funds and units
+///
+/// Provides imperative operations for observing funds, transferring funds and
+/// observing refunds of various units.
+/// The two currently support units are `#cycle` and `#icpt`, measuring
+/// computational cost and Internet Computer tokens, respectively.
```

Comment:
Cycles are not just for computational cost (also storage), and probably the costs will be _measured_ in something else that has a conversion rate to cycles. Or something. Timo points out that the term “cycle” is very overloaded. Maybe
```suggestion
/// The two currently support units are `#cycle` and `#icpt`. Usage of the Internet Computer
/// is charged for in Cycles. ICPTs are Internet Computer tokens.
```
hmm, also not great.


Maybe @lsgunnlsgunn phrase this, she probably has a better idea about our official terminology than me.

Reply:
Maybe just "A Cycle is the unit of Internet Computer utilization; ICPT are Internet Computer tokens."